neo4j/test_neo4j.py

```diff
@@ -0,0 +1,158 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+from nose.plugins.attrib import attr
+
+# project
+from checks import AgentCheck
+from utils.platform import Platform
+from shared.test.common import AgentCheckTest
+
+
+@attr(requires='mysql')
+class TestMySql(AgentCheckTest):
+    CHECK_NAME = 'neo4j'
+
+    METRIC_TAGS = ['tag1', 'tag2']
+    SC_TAGS = ['server:localhost', 'port:13306']
+    SC_FAILURE_TAGS = ['server:localhost', 'port:unix_socket']
+
+
+    MYSQL_MINIMAL_CONFIG = [{
+        'server': 'localhost',
+        'user': 'neo4j',
+        'pass': 'neo4j',
+        'port': '7474'
+    }]
+
+
+    CONNECTION_FAILURE = [{
+        'server': 'localhost',
+        'user': 'unknown',
+        'pass': 'dog',
+    }]
+
+    NEO4J_VARS = [
+        'kernelversion',
+	#        'databasename',
+        'mbeanquery',
+        'storeid',
+        'readonly',
+        'storecreationdate',
+        'storelogversion',
+        'kernelstarttime',
+        'lastcommittedtxid',
+        'peaknumberofconcurrenttransactions',
+        'numberofrolledbacktransactions',
+        'numberofopentransactions',
+        'numberofopenedtransactions',
+        'numberofcommittedtransactions',
```

Comment:
I'm not sure the number prefix adds much value here. Would it be more readable without that?